### PR TITLE
Update function arguments (from sklearn) in Representation.

### PR DESCRIPTION
**Pull request by henrifroese**
- Remove arguments that are unnecessarily complex for most users
- Add arguments that are useful to many users.
- Revise default argument values (n_jobs=-1 where possible, copy_x=True where possible)

We believe that the arguments that are available to users now make the most sense for the community (as users with very complex needs will probably use sklearn anyway).

### CODE REVIEW

**Comment by henrifroese** [comment]
Just removed the `verbose` argument

File: texthero/representation.py [hunk 1]
```diff
@@ -496,25 +504,17 @@ def nmf(s, n_components=2) -> pd.Series:
     :meth:`tfidf` to compute TF-IDF and :meth:`term_frequency` to compute term frequency
 
     """
-    nmf = NMF(n_components=n_components, init="random", random_state=0)
+    nmf = NMF(n_components=n_components, init="random", random_state=random_state,)
```
Comment (jbesomi) [inline texthero/representation.py:507]: In contrast with `PCA`, we are not copying, is that expected?

**Comment by jbesomi** [comment]
Super great! 🎉 🎉 

See the only comment below. By the way, should we keep the ":meth:`tfidf` to compute TF-IDF and :meth:`term_frequency` to compute term frequency"? Is this a bit out of scope maybe?

**Comment by henrifroese** [comment]
> should we keep the ":meth:tfidf to compute TF-IDF and :meth:term_frequency to compute term frequency"? Is this a bit out of scope maybe?

Makes sense, it doesn't really belong in the "see also" section. Removed it.

**Comment by jbesomi** [comment]
Great, thank you. Merged! 🎉 🍻

**Pull request merged**